src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, Colorful {
+    protected String color;
+
+    public Figure() {
+        this.color = "white";
+    }
+
+    public interface Drawable {
+        void draw();
+    }
+
+    public interface Colorful {
+        String getColor();
+    }
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public abstract double getArea();
+
+    public abstract void draw();

```

Comment:
This method is unnecessary if you If you're implementing an abstract class - interface will provide it.

Reply:
Not fixed